scrape_linkedin/CompanyScraper.py

```diff
@@ -29,14 +46,27 @@ def scrape(self, company, overview=True, jobs=False, life=False, insights=False)
             jobs_html = self.fetch_page_html('jobs')
         if insights:
             insights_html = self.fetch_page_html('insights')
-        return Company(overview_html, jobs_html, life_html, insights_html)
+        if people:
+            people_html = self.fetch_page_html('people')
+
+        return Company(overview_html, jobs_html, life_html, insights_html,
+                       people_html)
 
     def fetch_page_html(self, page):
         """
         Navigates to a company subpage and returns the entire HTML contents of the page.
         """
+
+        if page == "people":
+            interval = 2.0
+        else:
+            interval = 0.1
+
         try:
             self.driver.get(f"{self.url}/{page}")
+            # people/alumni javascript takes more time to load
+            time.sleep(interval)
+
```

Comment:
Instead of a hard-coded sleep, which can cause both unnecessary delays for people with fast internet, and false delays for those with slow internet, I'd instead suggest using a `WebDriverWait(self.driver, self.timeout).until(...)`, which you can see an example of in `load_initial`.